books/free-programming-books-he.md

```diff
@@ -0,0 +1,36 @@
+### Index
+
+* [C#](#cs)
+* [Assembly](#assembly)
+* [Python](#python)
+* [ללא תלות בשפה](#ללא-תלות-בשפה)
+  * [מערכות הפעלה](#מערכות-הפעלה)
+  * [רשתות](#רשתות)
+
+
+### Assembly
+
+* [ארגון המחשב ושפת סף](https://data.cyber.org.il/python/python_book.pdf) – ברק גונן, המרכז לחינוך סייבר (PDF)
+
+
+### <a name="cs">C#</a>
+
+* [מבוא לתכנות בסביבת האינטרנט בשפת C#](https://meyda.education.gov.il/files/free%20books/%D7%9E%D7%91%D7%95%D7%90%20%D7%9C%D7%AA%D7%9B%D7%A0%D7%95%D7%AA%20%D7%91%D7%A1%D7%91%D7%99%D7%91%D7%AA%20%D7%94%D7%90%D7%99%D7%A0%D7%98%D7%A8%D7%A0%D7%98%20090216.pdf) – מט״ח (PDF)
+
+
+### Python
+
+* [תכנות בשפת פייתון](https://data.cyber.org.il/python/python_book.pdf) – ברק גונן, המרכז לחינוך סייבר (PDF)
+
+
+### ללא תלות בשפה
+
+#### מערכות הפעלה
+
+* [מערכות הפעלה](https://data.cyber.org.il/os/os_book.pdf) – ברק גונן, המרכז לחינוך סייבר (PDF)
+
+
+#### רשתות
+
+* [רשתות מחשבים](https://data.cyber.org.il/networks/networks.pdf) – עומר רוזנבוים, ברק גונן, שלומי הוד, המרכז לחינוך  (PDF)סייבר
```

Comment:
Move to top with the purpose of mantaining list alphabetization proposed at TOC code hunk

If it's not possible, ignore it.

Reply:
Done by commit 9e1c741